ios/FluentUI/Controls/SearchBar.swift

```diff
@@ -214,6 +225,13 @@ open class SearchBar: UIView {
         return clearButton
     }()
 
+    //indicates search in progress
```

Comment:
needs to be apple documentation commenting style for clarity

Reply:
fixed

---
In reply to: [465866211](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r465866211) [](ancestors = 465866211)

Reply:
Thanks :)

---
In reply to: [465912220](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r465912220) [](ancestors = 465912220)